json-schema/json-schema/src/main/scala/endpoints4s/NumericConstraints.scala

```diff
@@ -0,0 +1,87 @@
+package endpoints4s
+
+/** Possible restrictions on the numeric value used. Needs an instance for [[Ordering]] to check whether values are
```

Comment:
```suggestion
/** Possible restrictions on the numeric value used. Needs an instance for `Ordering` to check whether values are
```

Seems like scaladoc does not know how to link to `Ordering`…